src/components/ATATFileUpload.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div v-cloak 
```

Comment:
General layout thoughts: Using the attached, Figma on left, SPA on right, some minor margins/paddings adjustments should be made to match Figma more precisely.
![Screen Shot 2022-05-04 at 6 22 45 PM](https://user-images.githubusercontent.com/90333349/166835359-a92bd906-f34f-4eaa-8d19-e1b6a5f0c0d5.png)

Reply:
- [x] aligning issues resolved. Figma on the left, web page on the right
![image](https://user-images.githubusercontent.com/84856468/166851355-f08342b3-b05b-425a-bff1-aa9aa3566c8d.png)

Reply:
looks good - I was just pointing out general spacing issues. this is much better!